### ISSUE TITLE
WCAG 2.1 AA Accessibility: Form cross-team working group

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#31240]
## Description

We want to ensure we find and fix as many accessibility issues **across the entire application**.

We should form a working group of "accessibility champions" that can help us do this.

This group should have:
- Own Slack channel, or use #accessibility 
- Own @ handle on Slack

**Comment by umpox** [comment]
Closed, we now have [#wg-accessibility-champions](https://sourcegraph.slack.com/archives/C038K3PR2M7)!
